lambdas/management/status.go

```diff
@@ -13,10 +12,9 @@ import (
 )
 
 type StatusRequest struct {
```

Comment:
This struct is only being used as input to the management lambda, right?
Lets rename it to `WekaStatusRequest` so it will be more clear.